vllm/model_executor/models/mixtral_quant.py

```diff
@@ -140,9 +140,10 @@ def forward(self, hidden_states: torch.Tensor) -> torch.Tensor:
         router_logits, _ = self.gate(hidden_states)
 
         routing_weights = F.softmax(router_logits, dim=1, dtype=torch.float)
-        routing_weights, selected_experts = torch.topk(routing_weights,
-                                                       self.top_k,
-                                                       dim=-1)
+        # Use optimized fast_topk for better performance when top_k=1
+        routing_weights, selected_experts = fast_topk(routing_weights,
+                                                      self.top_k,
+                                                      dim=-1)
```

Comment:
Mixtral seems to use top_k==2 since the config says `"num_experts_per_tok": 2`, so not sure this will see a benefit
https://huggingface.co/mistralai/Mixtral-8x7B-Instruct-v0.1/blob/main/config.json